qa/L0_python_api/testing_utils.py

```diff
@@ -93,3 +95,58 @@ def send_and_test_inference_identity(frontend_client, url: str) -> bool:
 
     teardown_client(client)
     return input_data[0] == output_data[0].decode()
+
+
+# Sends a streaming inference request to test_model_repository/identity model
```

Comment:
nit: personal preference is to follow Google style guide for docstrings&comments: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

Reply:
Added TODO to fix in future PR